docs/src/python/user-guide/expressions/numba-example.py

```diff
@@ -0,0 +1,17 @@
+import polars as pl
+import numba as nb
+
+df = pl.DataFrame({"a": [10, 9, 8, 7]})
+
+
+@nb.guvectorize([(nb.int64[:], nb.int64, nb.int64[:])], "(n),()->(n)")
+def cum_sum_reset(x, y, res):
+    res[0] = x[0]
+    for i in range(1, x.shape[0]):
+        res[i] = x[i] + res[i - 1]
+        if res[i] >= y:
+            res[i] = x[i]
+
+
+out = df.select(cum_sum_reset(pl.all(), 5))
+print(out)
```

Comment:
I'm confused by this example - for these particular numbers, they're all above the threshold, so wouldn't the result just be the same as the input? maybe run it with `cum_sum_reset(pl.all(), 30)`?